Templates/SaaSFileTrigger-Batch/metadata.json

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "SaaSFileTrigger - Batch",
+    "description": "A Batch function that will be run whenever a file is added to a SaaS provider.",
```

Comment:
That's a separate issue we are tracking. https://github.com/Azure/azure-webjobs-sdk-extensions/issues/96

You may add this comment there.